.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -497,7 +497,7 @@ jobs:
       JAVA_VER: 17
       PLUGIN_BUILD: ${{ github.workspace }}/plugin-build
       PLUGIN_SOURCE: ${{ github.workspace }}/plugin-source
-    if: github.ref == 'refs/heads/nightly' || contains(github.ref, 'release/') || inputs.knime
+    if: (github.ref == 'refs/heads/nightly' || contains(github.ref, 'release/') || inputs.knime) && !startsWith(steps.set-vars.outputs.cxx_compiler, 'clang++') 
```

Comment:
- please documentation 
- explicitly specifying the build matrix with || (don't use negation / exclusion) e.g.  ... && (comp1 || comp2 || comp3)